HPXMLtoOpenStudio/measure.rb

```diff
@@ -1625,6 +1636,13 @@ def self.add_setpoints(runner, model, weather, spaces)
     hvac_control = @hpxml.hvac_controls[0]
     living_zone = spaces[HPXML::LocationLivingSpace].thermalZone.get
     has_ceiling_fan = (@hpxml.ceiling_fans.size > 0)
+    timestep = @hpxml.header.timestep
+    # Do not apply on off thermostat if timestep is >= 2
+    # Only availabe with 1 min time step
+    if timestep >= 2 && hvac_control.onoff_thermostat_deadband > 0.0
+      hvac_control.onoff_thermostat_deadband = 0.0
+      runner.registerWarning('Time step too large to enable on-off thermostat deadband. Continute without on-off control.')
```

Comment:
It looks like you've added this same warning to EPvalidator.xml, so it can be removed from here?

Reply:
Also, how about moving this code to hpxml_defaults.rb so that the deadband gets reset to zero as early in the measure as possible? It would also then indicate in the `in.xml` whether we have overridden the value or not.